src/i18n/de.yaml

```diff
@@ -184,6 +184,7 @@ search:
   writeToSearch: Hier schreiben zum Suchen
 languages:
   en: English
+  it: Italian
```

Comment:
We're keeping the name of the language original. Could you just change it to 'Italiano' in all translations? 😄